api/journals/handler.go

```diff
@@ -0,0 +1,89 @@
+package journals
+
+import (
+	"github.com/go-playground/validator"
+	"github.com/labstack/echo/v4"
+	"github.com/matteoaricci/jot-api/models/journal"
+	"github.com/matteoaricci/jot-api/service/journal"
+	"net/http"
+)
+
+func AddRoutes(e *echo.Echo) {
+	g := e.Group("/api/journals")
+
+	g.GET("", func(c echo.Context) error {
```

Comment:
thought: this is totally my opinion and not common practice - but personally I try to never use API endpoint groups:
 - mainly because whenever I'm troubleshooting something and need to find the endpoint, I'm going to want to be able to search for the route
 - it's also much clearer what endpoint you are looking at when the route is right there, instead of `GET("",...`

I know it's very common to group endpoints like this (in all languages) but I think it's people going too far with DRY.

Ok end rant.